frontend/src/pages/ConfirmationOfReceiptPage/ReceivedByInput.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState, useEffect } from "react";
+import classNames from "classnames";
+import { Row, Cell } from "react-table";
+import { Button } from "solo-uswds";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faSpinner } from "@fortawesome/free-solid-svg-icons";
+import { DocumentWithReceivedBy } from "./tableColumns";
+import classes from "./ReceivedByInput.module.scss";
+
+interface ReceivedByInputCellProps {
+  onSubmitCOR: (sdn: string, receivedBy: string) => void;
+  cell: Cell<DocumentWithReceivedBy>;
+  row: Row<DocumentWithReceivedBy>;
+}
+
+const ReceivedByInputCell: React.FC<ReceivedByInputCellProps> = ({
+  onSubmitCOR,
+  row: {
+    original: { sdn, submitting, error }
+  }
+}) => {
+  const [receivedBy, setReceivedBy] = useState("");
+  const onSubmit: React.FormEventHandler<HTMLFormElement> = e => {
+    e.preventDefault();
+    onSubmitCOR(sdn, receivedBy);
+  };
+
+  useEffect(() => {
+    setReceivedBy("");
```

Comment:
Should this be `sdn` instead of `""`? Otherwise the dep array doesn't need `sdn`